### ISSUE TITLE
Can exSTRa identify STRs in non-human species?

### ISSUE THREAD

**Issue by zhizhanwang** [bahlolab/exSTRa#88]
hello， 
If I want to build a database for non-human species, what files should I prepare?
Thanks!

**Comment by mfbennett** [comment]
Yes, it is possible to use exSTRa for non-human species. You will need to prepare a database file that contains columns with information for each repeat locus that you wish to analyse. We provide a database file containing known, disease-causing RE in humans. (Note that many of the columns are optional and provided for information, but not required.)

Depending on the organism band whether you already have a set of STRs of interest, you may need to start by running Tandem Repeats Finder on the reference genome to generate a catalog of STRs. Once you have a list of STRs that you want to target, you can then create a database with columns containing chromosome, start/end position, repeat motif, etc, for each STR.

There is an example script for creating a database from the UCSC simpleRepeat track for humans (`inst/tools/prepare_exSTRa_input_db.R`), which may be helpful.

Let us know if you have any questions
